File: app/src/main/java/com/six/cat/sixcat/fragment/ThemeFragment.java
package com.six.cat.sixcat.fragment;

import android.os.Bundle;

import com.six.cat.sixcat.R;
import com.six.cat.sixcat.base.BaseRxLazyFragment;

/**
 * @author liguoying
 * @date 2017/12/11.
 */

public class ThemeFragment extends BaseRxLazyFragment {

    public static ThemeFragment newInstance() {
        return new ThemeFragment();
    }

    @Override
    public int getLayoutResId() {
        return R.layout.fragment_home;
    }

    @Override
    public void finishCreateView(Bundle state) {

    }
}

File: app/src/main/java/com/sixcat/views/fragment/ThemeFragment.kt
package com.sixcat.views.fragment

import com.sixcat.R
import com.sixcat.base.BaseRxLazyFragment

/**
 * @author liguoying
 * @date 2017/12/11.
 */

class ThemeFragment : BaseRxLazyFragment() {

    override fun getLayoutResId(): Int {
        return R.layout.fragment_home
    }


    override fun initView() {

    }

    override fun lazyLoad() {

    }

    companion object {

        fun newInstance(): ThemeFragment {
            return ThemeFragment()
        }
    }
}
